### ISSUE TITLE
Themes

### ISSUE THREAD

**Issue by Norien** [Norien/Home-Assistant-Config#116]
Thanks to @skalavala for the idea

- Change Theme based on sun
- Holiday based themes
- Change doorbell tone to match theme
- Change front porch light to match theme

Theme package started - https://github.com/Norien/Home-Assistant-Config/blob/master/packages/theme.yaml

**Comment by Norien** [comment]
looking for some way to display if current day is a holiday and what that holiday is.

**Comment by Norien** [comment]
@ccostan to the rescue once again - https://github.com/CCOSTAN/Home-AssistantConfig/blob/master/script/monthly_color_scene.yaml

**Comment by Norien** [comment]
@skalavala
https://blog.sparksuite.com/json-list-of-holidays-1eddc0ef48e0
https://raw.githubusercontent.com/skalavala/smarthome/master/holidays.json
```
- platform: rest
  scan_interval: 86400
  resource: http://ip.jsontest.com
  name: External IP
  value_template: '{{ value_json.ip }}'
```
`{{ holidays_json.MAJOR_US.static['1/1'] }}`
if that returns empty, you acess `{{ holidays_json.MAJOR_US.dynamic['1/1/2018'] }}`

**Comment by Norien** [comment]
sensor:
  - platform: rest
    resource: https://raw.githubusercontent.com/skalavala/smarthome/master/holidays.json
    name: Holiday
    value_template: >
      {% set today = now().month  ~ '/' ~ now().day  %}
      {% set tmp =  value_json.MAJOR_US.static[ today ] %}
      {% if tmp | trim == "" %}
        {% set today = now().month  ~ '/' ~ now().day ~ '/' ~ now().year %}
        {% set tmp =  value_json.MAJOR_US.dynamic[ today ] %}
      {% endif %}
      {{ tmp }}

**Comment by skalavala** [comment]
Hey Norien,

Here is the holiday data `https://github.com/skalavala/smarthome/blob/master/holidays.json`
and here is the package: `https://github.com/skalavala/smarthome/blob/master/packages/holidays.yaml`

We will see if we can have folks populate data for various countries, and possibly write a program that generates based on some logic.... those would be the next steps.

**Comment by Norien** [comment]
Thanks @skalavala, @CCOSTAN
Looks like I got everything working okay. I did have to change up the ```holiday.json``` to suit my needs here in Canada a little better and also output ```Standard``` if not a holiday just for use in a secondary sensor.

https://github.com/Norien/Home-Assistant-Config/blob/master/packages/theme.yaml
https://github.com/Norien/Home-Assistant-Config/blob/master/packages/holiday.yaml
https://github.com/Norien/Home-Assistant-Config/blob/master/www/holiday.json

- Theme is set on ```event: start``` to match ```input_select.theme```

- ```Frontend``` theme is switched for light and dark on ```sun.sun``` unless it's a holiday which the corresponding theme will be set. Themes are changed based off  ```input_select.theme```

- Front porch light is set to bright white ```[255,255,255]``` on ```event: sunset``` unless ```sensor.holiday != Standard``` which it will grab the ```rgb_color``` based on what the ```input_select.theme``` is set. Light is turned off once ```script.good_night``` is activated.
